django/conf/global_settings.py

```diff
@@ -310,6 +310,15 @@
 # file system instead of into memory.
 FILE_UPLOAD_MAX_MEMORY_SIZE = 2621440  # i.e. 2.5 MB
 
+# Maximum size in bytes of request data that will be read before raising a
+# SuspiciousOperation. This applies to accessing request.body and also to any
+# data in form data requests, excluding file uploads.
+DATA_UPLOAD_MAX_MEMORY_SIZE = 2621440  # i.e. 2.5 MB
+
+# Maximum number of keys in a POST request that will be allowed before raising
+# a SuspiciousOperation.
+DATA_UPLOAD_MAX_NUMBER_FIELDS = 100
```

Comment:
Dunno, I do not have hard data to prove it :D I know that 100 will be fine for most of my sites and on some others I'll completely disable this…